src/Sulu/Bundle/ContentBundle/Content/Structure/ExcerptStructureExtension.php

```diff
@@ -90,7 +90,10 @@ public function __construct(
      */
     public function save(NodeInterface $node, $data, $webspaceKey, $languageCode)
     {
-        foreach ($this->excerptStructure->getProperties() as $property) {
+        $excerptStructure = $this->getExcerptStructure();
```

Comment:
Yeah, I would always get a new instances, sounds safer, and we have less state on this class. Although there is still too much state saved in the `AbstractExtension`, but I guess that should be part of a bigger refactoring.